spec/policies/form_policy_spec.rb

```diff
@@ -26,20 +26,24 @@
   end
 
   describe "#can_view_form?" do
-    context "with a form editor" do
-      it { is_expected.to permit_actions(%i[can_view_form]) }
+    (User.roles.keys - %w[trial]).each do |role|
```

Comment:
Is it worth adding a class constant to the User model to do this for us?

Something like:
```ruby
APPROVED_ROLES = [:editor, :super_admin]
```